### PR TITLE
Builds out the Goldify Landing Page

### LINKED ISSUE #23: Build Landing Page
**Is your feature request related to a problem? Please describe.**
There is currently no landing page, besides some basic text.

**Describe the solution you'd like**
Ideally, this is done in conjunction with #3, since this is essentially the same bit here.

**Describe alternatives you've considered**
N/A

**Additional context**
I think we could maybe just pull the README.md from our repo's root dir and just show that on our Landing Page. To me, it's easy to read, and kinda fun for developers as well.

### PR DESCRIPTION
**Pull request by MattGrantUMich**
- Includes brief description/goal of the application
- Includes some icons describing a few of the added elements (i.e. mobile friendly)
- Includes a Q&A section with some useful information

Definitely open to updates here.

Closes #23

### CODE REVIEW

**Comment by MattGrantUMich** [comment]
Also my apologies I didn't mean to push the commit here before running the tests.
That won't happen again.

File: src/js/landing/GoldifyLandingPage.jsx [hunk 1]
```diff
@@ -8,11 +16,97 @@ class GoldifyLandingPage extends Component {
...
+            hits to build your personal playlist from. Within minutes, you will
+            have a Spotify playlist that is molded exactly to your musical
+            taste. We&apos;re excited for you to be here and can&apos;t wait for
+            you to enjoy your golden tracks!
+          </p>
+          <div className="landing-feature-icons">
+            <div className="landing-feature-icon">
+              <MobileFriendlyIcon style={{ fontSize: 40, color: grey[900] }} />
+              <br />
+              <h4>Mobile Friendly</h4>
```
Comment (kyrod) [inline src/js/landing/GoldifyLandingPage.jsx:50]: 🤧

File: src/js/landing/GoldifyLandingPage.jsx [hunk 2]
```diff
@@ -8,11 +16,97 @@ class GoldifyLandingPage extends Component {
...
+            </div>
+            <div className="landing-feature-icon">
+              <VerifiedUserIcon style={{ fontSize: 40, color: grey[900] }} />
+              <br />
+              <h4>Clean User Interface</h4>
```
Comment (kyrod) [inline src/js/landing/GoldifyLandingPage.jsx:55]: 🤧

File: src/js/landing/GoldifyLandingPage.jsx [hunk 3]
```diff
@@ -8,11 +16,97 @@ class GoldifyLandingPage extends Component {
...
+            </div>
+            <div className="landing-feature-icon">
+              <FastForwardIcon style={{ fontSize: 40, color: grey[900] }} />
+              <br />
+              <h4>Instant Updates</h4>
```
Comment (kyrod) [inline src/js/landing/GoldifyLandingPage.jsx:60]: 🤧

**Review by kyrod — APPROVED** [review]
Looks good! We may want to revisit this again when we do the Readme, but this is great for now!

**Pull request merged**